Robyn/dollar

robyng opened the pull request:
Need help to display correct amount of dollar signs. And weird thing happens when I press heart button it repeats dollar signs!

robyng committed:
add to app.css dollar styles.

robyng committed:
started styled component for DollarStyle

robyng committed:
add test, remove redyeti from styled

robyng committed:
remove old css

robyng committed:
remove old comments, add priceLevel function loop.
add to useEffect setPrice.

robyng commented:
Working now!

robyng committed:
Route sign up page

robyng committed:
sign up export

robyng committed:
remove unused Button, breaking change

robyng committed:
Add sign up and login to nav, add breaks

johndavis92790 commented on client/src/pages/SearchResult/SingleSearchResult.tsx:
```diff
@@ -47,6 +54,8 @@ export function SingleSearchResultPage() {
       try {
         const data = await getRestaurant(place_id);
         setRestaurantData(data.result);
+        setPrice(data.result.price_level);
```
We don't need line 57 anymore since its being set on line 73 from the restaurantData object

johndavis92790 commented on client/src/pages/SearchResult/SingleSearchResult.tsx:
```diff
@@ -47,6 +54,8 @@ export function SingleSearchResultPage() {
         const data = await getRestaurant(place_id);
         setRestaurantData(data.result);
+        setPrice(data.result.price_level);
+        // add heart api later
```
Please add a TODO to the beginning of a comment like this so make it easier to find these types of comments later through search.

johndavis92790 commented on client/src/pages/SearchResult/SingleSearchResult.tsx:
```diff
@@ -60,6 +69,18 @@ export function SingleSearchResultPage() {
   const prevState = "";
+  // dollar
+  let dollarAPI = restaurantData.price_level;
+  console.log("dollarAPI: " + dollarAPI);
```
Please remove `console.log`s that don't return errors before merging into develop.

johndavis92790 commented on client/src/pages/SearchResult/SingleSearchResult.tsx:
```diff
@@ -60,6 +69,18 @@ export function SingleSearchResultPage() {
+  let dollarAPI = restaurantData.price_level;
+  console.log("dollarAPI: " + dollarAPI);
+
+  const [price, setPrice] = useState(0);
```
We don't need line 76 anymore since its being set on line 73 from the restaurantData object

johndavis92790 commented on client/src/pages/SearchResult/SingleSearchResult.tsx:
```diff
@@ -71,17 +92,10 @@ export function SingleSearchResultPage() {
               <HeartIcon src={heartFilled} />
```
Why are we using `prevState` here, rather than using the `useState` boolean from line 70 directly? You could add a function to toggle `heart`, if `heart` is false and you click on this button, it will `setHeart` to true, and vice versa.

robyng replied:
Removed. I forget what I was trying to do here.

johndavis92790 reviewed (changes requested):
Getting close!

robyng committed:
Merge pull request #129 from Healthy-Plate-Project/robyn/login-nav

Robyn/login nav

johndavis92790 committed:
Get Google Place Photo
- Add getGooglePhoto() server call. Set testPhotoData = true which will use the GoogleTestPhoto.jpg, rather than using up our Google API account.
- Add GoogleTestPhoto.jpg
- Add test in SingleSearchResult.tsx to give example of how it can be used.
- Add photoRoutes.js, photoController.js for handling post request from front end. Uses Google API key stored in the server env file to create URL and sends it back to front end which gives the correct URL for the photo to be displayed.

johndavis92790 committed:
Fix testAPI boolean for using test data, rather than actual Google API calls.
- All you need to do to switch between the two is to change exports.testAPI = true in the server index.js fil to false for Google data, and true for test data.

MatteoThomas committed:
Merge pull request #136 from Healthy-Plate-Project/john/photos-api-call

Get Google Place Photo

robyng committed:
add TODO and remove console logs and unused code from Johns review requests

robyng committed:
Merge remote-tracking branch 'origin' into robyn/dollar
